clang/utils/perf-training/lit.site.cfg.in

```diff
@@ -11,6 +11,7 @@ config.python_exe = "@Python3_EXECUTABLE@"
 config.cmake_exe = "@CMAKE_COMMAND@"
 config.llvm_src_dir ="@CMAKE_SOURCE_DIR@"
 config.cmake_generator ="@CMAKE_GENERATOR@"
+config.use_llvm_build = @CLANG_PGO_TRAINING_USE_LLVM_BUILD@
```

Comment:
`@CLANG_PGO_TRAINING_USE_LLVM_BUILD@` needs to be quoted.  This is causing  a build failure in the Release build configuration.

Reply:
Should be addressed by #127967.